fixed_model.config.in

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<model>
+    <name>@ROBOT_NAME@_fixed (no hands)</name>
```

Comment:
That `(no hands)` is not ideal, but I see that you got it from the existing `model.config.in`,  so if we remove it we will remove it from both.